### DISCUSSION TITLE
How do i add info in 7.4 auth

### DISCUSSION THREAD

**Discussion by Fall0n** [UltimaHoarder/UltimaScraper#1378] [Q&A]
Its not the same way that 7.3 

hope some one can make lite auth file that have "not working info in it" and show me.. Or if i can make auth why i think and some one can tell me if its wrong..

**Comment by OFfriend** [comment]
you can use the example from the readme file 

![grafik](https://user-images.githubusercontent.com/83786268/122647346-5bd1ec00-d113-11eb-8f5a-0c5205782caa.png)

**Reply by mediaburnwayne** [reply]
Yeah I am not able to make this work. Got to be missing something Here is minw with the vaues changed or modified for privicy. Is it a formatting issue

{
  "auth": {
    "username": "default",
    "cookie": "auth_id=00000000; sess=bkja17b623snntk7fo3oannk1u; auth_hash=b0c6e7813d14602049ca1g185a16edac; auth_uniq_00000000=; auth_uid_00000000=;",
    "user_agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.106",
    "email": "myemail@gmail.com",
    "password": "mypassword123",
    "hashed": false,
    "support_2fa": false,
    "active": true
  }
}

**Comment by OFfriend** [comment]
copy the entire cookie like described in the readme. `auth_id=00000000` , `auth_uniq_00000000=` and `auth_uid_00000000=` is wrong.

![grafik](https://user-images.githubusercontent.com/83786268/122648999-4791ed00-d11b-11eb-9f92-bb083e41220e.png)

**Reply by mediaburnwayne** [reply]
Copying entire cookie and placing in between doublequotes in auth.json  does not work for me, it just runs through the 10 failures

**Reply by OFfriend** [reply]
just copy the "sess= ... " part without `cookie:`

**Reply by OFfriend** [reply]
you use 2FA?

**Reply by mediaburnwayne** [reply]
No have it set to false. I just whacked it and I and I see there are 5 values. for cookie, I have populated all of them except for auth_uniq, the rest looks like this

 "user_agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.106 Safari/537.36",
    "email": "",
    "password": "",
    "hashed": false,
    "support_2fa": false,
    "active": true

**Reply by mediaburnwayne** [reply]
I deleted everything and rebuilt all files, and it seems to be working

**Reply by Fall0n** [reply]
still not working for me.. Copy the cookie and its not working done likte its show on webpage now and do tell me in text file. My cookie file do not have the sess.

**Reply by UltimaHoarder** [reply]
If you're using 2fa turn it off.

**Reply by fightcancer2** [reply]
> 
> 
> If you're using 2fa turn it off.

Does that suggestion apply to most everyone?

I set "support_2fa" to false and I've tried deleting that line entirely from auth.json, but I keep getting this error:

"Auth (V1) Attempt 10/10
Access denied.
Traceback (most recent call last):
  File "G:\GitHub\OnlyFans-7.4\start_ofd.py", line 62, in <module>
    loop.run_until_complete(main())
  File "C:\Users\xxxxxxxx\AppData\Local\Programs\Python\Python39\lib\asyncio\base_events.py", line 642, in run_until_complete
    return future.result()
  File "G:\GitHub\OnlyFans-7.4\start_ofd.py", line 46, in main
    api = await main_datascraper.start_datascraper(
  File "G:\GitHub\OnlyFans-7.4\datascraper\main_datascraper.py", line 88, in start_datascraper
    main_helper.export_data(auth_details, user_auth_filepath)
  File "G:\GitHub\OnlyFans-7.4\helpers\main_helper.py", line 771, in export_data
    ujson.dump(metadata, outfile, indent=2, escape_forward_slashes=False)
TypeError: <apis.onlyfans.classes.extras.cookie_parser object at 0x000002170FFA0F40> is not JSON serializable

Then my auth.json gets blanked out so it's a 0-kb file, and I have to login to OF on the web and get a new sess, update my auth.json, etc.

**I'm glad to pay someone for help**: BTC, Pp or buying someone a cheap OF sub and uploading to Mega once I get this wonderful app working again.

Here's my auth.json file with a slightly edited sess and auth_id.

{
  "auth": {
    "username": "default",
    "cookie": "auth_id=xxxxxx466; sess=8lstdsjkv5bbf686sep2c5ptql; auth_hash=; auth_uniq_xxxxxx466=; auth_uid_xxxxxx466=;",
    "user_agent": "User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:89.0) Gecko/20100101 Firefox/89.0",
    "email": "",
    "password": "",
    "hashed": false,
    "support_2fa": false,
    "active": true
  }
}

**Reply by UltimaHoarder** [reply]
@fightcancer2 try
https://github.com/DIGITALCRIMINAL/OnlyFans/issues/1413

**Reply by fightcancer2** [reply]
I tried it, but it still isn't working for me.  I'm using an older version of the scraper for now.

**Reply by lucky1luc** [reply]
use another browser (like MS Edge) follow the same instructions, make sure your cookie line ends with ", (double quote + comma) don't forget to copy the user agent into the auth.json as well
